EnglishTranslation-master/QUEST.tsv

```diff
@@ -3610,246 +3610,246 @@ QUEST_20160224_003617	I don't think I'll be able to ask for any more help today.
-QUEST_20160310_003729	Seems you don't have any new stories to tell. {nl}Come back if you get to explore any new areas.
-QUEST_20160310_003730	Because of the monsters we can't do anything the way we want.{nl}We used to go to Siauliai Woods to pick herbs, eat fruit from the trees or get some rest, but not anymore.
+QUEST_20160310_003729	It looks like you don't have any new stories to tell. {nl}Come back if you get to explore any new areas.
+QUEST_20160310_003730	Because of the monsters we can't do anything the way we want.{nl}We used to go to Siauliai Woods to pick herbs, eat fruit from the trees or get some rest, {nl}but not anymore.
 QUEST_20160310_003731	Living in a place with these monsters running outside is driving me insane.{nl}But because of them I can't even dare to move somewhere else...
-QUEST_20160310_003732	Lately I can't so much as get a good night of sleep.{nl}All because of the monsters.
+QUEST_20160310_003732	Lately I can't do so much as get a good night of sleep.{nl}All because of the monsters.
 QUEST_20160310_003733	Manual
 QUEST_20160310_003734	Medicine on top organized from brightest to darkest color.{nl}Red medicine on the left contains dangerous ingredients.{nl}Ingredients at the bottom organized from heaviest to lightest.{nl}Report missing ingredients on the corresponding sheet.{nl}For other questions, contact Sarma.{nl}{S30} - Sarma, Head of Research -{/}{nl}
 QUEST_20160310_003735	Before you read this, please make sure no one else is around.{nl}I'm Gatre and I'm helping Sarma with his experiments.{nl}Sarma approached me saying he wanted to do research to get the forest's fruit trees back to normal.{nl}
 QUEST_20160310_003736	We had lost a place to live after the trees grew too big and we couldn't eat the fruits, so Sarma's proposal appealed to us.{nl}He offered to share food with whoever helped him so most people from my village ended up accepting.{nl}
 QUEST_20160310_003737	At first we worked hard to help him save our hometown.{nl}But then Sarma's research started to feel somewhat strange.{nl}It seems like Sarma's research has little to do with getting the overgrown trees back to normal.{nl}The experiments are mostly done on monsters, not on trees, and I've even fallen unconscious during one experiment.{nl}
-QUEST_20160310_003738	I suspect Sarma is doing this to cover up for something evil.{nl}But because I have no proof, there's really nothing I can do.{nl}I know I can't ask you to trust my every word.{nl}But at least I want to know if my suspicions are correct, or if I'm simply overthinking it.{nl}
+QUEST_20160310_003738	I suspect Sarma is doing this to cover up for something evil.{nl}But because I have no proof, there's really nothing I can do.{nl}I know I can't ask you to trust every single word of mine.{nl}But at least I want to know if my suspicions are correct, or if I'm simply overthinking it.{nl}
```

Comment:
"something evil.{nl}But because I have no proof"
  `something evil.{nl}Since I have no proof`
"I can't ask you to trust every single word of mine.{nl}But at least I want to know if my suspicions are correct, or if I'm simply overthinking it."
 `I can't ask you to trust every single word of mine,{nl}but I at least want to know if my suspicions are correct or if I'm simply overthinking it.`